nix/lib/default.nix

```diff
@@ -139,8 +140,8 @@ in rec {
         source = "https://github.com/mozilla/release-services";
         commit = githubCommit;
         build =
-          if taskGroupId != ""
-            then "https://tools.taskcluster.net/groups/${taskGroupId}"
+          if taskGroupId != "" || taskId != ""
```

Comment:
That seems unrelated to the timeout

Reply:
yes, unrelated, probable i should call this PR "minor cloudops issues".

Reply:
and this should probably be `&&` instead of `||`

Reply:
we need this since cloudops added a check recently which checks in logs of a task to see if there is a correct docker image digest. without this automatic deployment to production is not possible.